lib/assets/javascripts/src/local-time/helpers/strftime.coffee

```diff
@@ -2,6 +2,16 @@ import LocalTime from "../local_time"
 
 {getI18nValue, translate} = LocalTime
 
+supportsIntlDateFormat = typeof Intl?.DateTimeFormat is "function"
+
+knownEdgeCaseTimeZones =
+  "Central European Summer Time": "CEST"
+  "Central European Standard Time": "CET"
+  "Western Indonesia Time": "WIB"
+  "Singapore Standard Time": "SGT"
+  "Москва, стандартное время": "MSK" # Europe/Moscow
+  "中国标准时间": "CST" # Asia/Shanghai
```

Comment:
This is the name of the timezone as returned in my javascript console. 

Notice how some time zone names in this list are in english, while others aren't. I just realized this might be due to only some of my chrome sensor locations including language locale information. The locations that don't include a locale are probably defaulting to english.

I'll test this out tomorrow. But this might mean that the browser's language might interfere with matching on these names.

I hope I'm wrong. And if I'm not then I hope there's a reliable way to get the time zone name in the same language every time.

Reply:
Did a super quick test that I'll need to validate later but this seems fixable!
![CleanShot 2023-08-21 at 11 35 46](https://github.com/basecamp/local_time/assets/31393016/66f36a80-54fc-4109-8b7a-0f4362aed2bc)

Reply:
Ohh, that's super nice, good catch!

Reply:
Ohh, that's really good for detecting the special cases, great catch!

Reply:
This seems to have worked!